golang/main.go

```diff
@@ -45,10 +47,11 @@ func bridgeToken(w http.ResponseWriter, r *http.Request) {
 
 // verifications accepts requests for a verification and sends the response
 func verifications(w http.ResponseWriter, r *http.Request) {
+	var err error
 	productType := os.Getenv("API_PRODUCT_TYPE")
 	splitPath := strings.Split(r.URL.Path, "/")
 	token := splitPath[2]
-	accessToken, err := getAccessToken(token)
+	accessToken, err = getAccessToken(token)
```

Comment:
why do we need global `accessToken`?
let's define it locally?
```
accessToken, err := getAccessToken(token)
```

Reply:
global is needed because it is used by the refresh. If we define locally then on the refresh calls we won't have access to it

Reply:
does it mean that `accessToken` can be overwritten by another user's request? 
how to deal with parallel requests from different users?

Reply:
there won't be parallel requests. This is a simple quickstart meant to show users how the product works